pkgs/by-name/ma/marwaita/package.nix

```diff
@@ -10,13 +10,13 @@
 
 stdenvNoCC.mkDerivation rec {
   pname = "marwaita";
-  version = "20.1";
+  version = "20.2-unstable-2024-07-01";
```

Comment:
it seems this has been tagged [here](https://github.com/darkomarko42/Marwaita/releases/tag/20.2). this applies to #323893, #323798, and #323801 as well

Reply:
I am including some fixes that were added upstream just after the tag was created. The releases versions in pling include them.